easybuild/easyconfigs/p/pstoedit/pstoedit-3.78-GCC-11.3.0.eb

```diff
@@ -0,0 +1,50 @@
+# Updated version from the original one.
+# Includes a patch so ImageMagick can be found, some missing
+# dependencies included as well. 
+# libemf seems to be needed.  
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'pstoedit'
+version = '3.78'
+
+homepage = 'http://pstoedit.net/'
+description = "pstoedit translates PostScript and PDF graphics into other vector formats"
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCE_TAR_GZ]
+patches = [('pstoedit-configure.patch')]
+
+checksums = [
+    '8cc28e34bc7f88d913780f8074e813dd5aaa0ac2056a6b36d4bf004a0e90d801',  # pstoedit-3.78.tar.gz
+    'c81588eebf9c1dd344566654aae8e826b013c969057cffd8e2cc2cd0a93b90c3',  # pstoedit-configure.patch
+]
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+    ('binutils', '2.38'),
+    ('gawk', '5.1.1'),
+    ('Bison', '3.8.2'),
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('ImageMagick', '7.1.0-37'),
+    ('Zip', '3.0'),
+    ('libzip', '1.9.2'),
+    ('Ghostscript', '9.56.1'),
+]
+
+osdependencies = [('libemf-dev', 'libEMF-devel', 'libplot-dev')]
```

Comment:
I don't see a reason not to create easyconfigs for these:
https://sourceforge.net/projects/libemf/
https://ftp.gnu.org/gnu/plotutils/

Reply:
I second that.

Reply:
And I done that. Now you just have a few more PRs open. :-)
For the plotutils: that is quite old code and not really maintained. I am wondering if that is a good idea to have it in there, or if there is a replacement for that as well.